build.gradle

```diff
@@ -913,7 +913,7 @@ project('test:system') {
 
 def getProjectVersion() {
     String ver = pravegaVersion
-    if (ver.contains("-SNAPSHOT")) {
+    if (grgit && ver.contains("-SNAPSHOT")) {
```

Comment:
One clarification: where is `grgit` defined? Is it defined as part of loading the plugin?

Reply:
yes, grgit is defined as part of loading the plugin.